### PR TITLE
feat(sso-auth): Implemented the SSO Auth in the header component (SSI)

### PR DESCRIPTION
**Pull request by deshmukhmayur**
# Closes JIRA issue 13

### 

# Explain the feature/fix

Implemented SSO auth and the user - profile drawer;
Also,
* integrated App Drawer with the Home Service API for getHomeTypeBy
* fixed some dependencies upgrades
* updated the docker-compose.yml
* updated README.md to include steps to use docker-compose for local development

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

* User Profile Drawer
Screenshot 2020-05-18 at 6 14 15 PM

* App Drawer (after integrating it with Home Service API
Screenshot 2020-05-18 at 6 14 27 PM

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by deshmukhmayur** [comment]
Oh wait, found a bug. Converting this to a draft.

**Comment by deshmukhmayur** [comment]
Ready for review.

File: packages/ssi-service/jest.config.js [hunk 1]
```diff
@@ -134,8 +134,7 @@ module.exports = {
   // snapshotSerializers: [],
 
   // The test environment that will be used for testing
-  // WKRD: Using jsdom-sixteen as it has support for customElements()
-  testEnvironment: "jest-environment-jsdom-sixteen",
+  // testEnvironment: "jest-environment-jsdom",
```
Comment (riginoommen) [inline packages/ssi-service/jest.config.js:137]: Hope this is because of the past package upgrade.

Comment (deshmukhmayur) [reply packages/ssi-service/jest.config.js:137]: Yes.

**Pull request merged**